### PR TITLE
Feature/restaurant

### PR DESCRIPTION
**Pull request by JAAR91**
Restaurant Page
Project includes:
-Home page
-Menu
-Contact Us

### CODE REVIEW

**Review by NtwaliHeritier — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUESTED🛑️🛑️🛑️🛑️**

Hello @JAAR91 

Great work on this project. You made a great job on this project. The app looks fantastic and remarkable. Great job👏️👏️

Let's make small changes and get it approved

[REQUIRED]: Differentiate content of tabbed pane in different modules.

[OPTIONAL]: It would be good to place images in a folder for better organization.

[REQUIRED]: Add steps to run the project in the readme file.

[REQUIRED]: Add this line `"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
` to package.json to run project with npm start instead of double click on index.html

That's it

Make the required changes and ask for another review👍️👍️

Happy coding💻️💻️💻️

File: README.md
Comment (rukundoeric) [inline README.md]: This title is not for this project, kindly change it before you merge.

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - Follows gitwork-flow.  💯
  - Codebase looks clean and professional. 💯 
  - All issues raised from the previous review are addressed. 💯 


- ### To Improve
  - The title  in README is not for this project, kindly make sure to change it before you merge this PR.

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/),
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**
